ui/admin/app/components/form/credential/username_password/index.hbs

```diff
@@ -78,42 +85,50 @@
     </InfoField>
   {{/if}}
 
-  <form.input
-    @name='username'
+  <Hds::Form::TextInput::Field
+    name='username'
     @value={{@model.username}}
-    @label={{t 'resources.credential.form.username.label'}}
-    @helperText={{t 'resources.credential.form.username.help'}}
-    @error={{@model.errors.username}}
-    as |field|
+    @isInvalid={{@model.errors.username}}
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'username')}}
+    as |F|
   >
+    <F.Label>{{t 'resources.credential.form.username.label'}}</F.Label>
+    <F.HelperText>{{t 'resources.credential.form.username.help'}}</F.HelperText>
     {{#if @model.errors.username}}
-      <field.errors as |errors|>
+      <F.Error as |E|>
         {{#each @model.errors.username as |error|}}
-          <errors.message>{{error.message}}</errors.message>
+          <E.Message>{{error.message}}</E.Message>
         {{/each}}
-      </field.errors>
+      </F.Error>
     {{/if}}
-  </form.input>
+  </Hds::Form::TextInput::Field>
 
   {{#if (or @model.isNew form.isEditable)}}
-    <form.input
-      @name='password'
-      @type='password'
+    <Hds::Form::TextInput::Field
+      name='password'
       @value={{@model.password}}
-      @label={{t 'resources.credential.form.password.label'}}
-      @helperText={{t 'resources.credential.form.password.help'}}
-      @error={{@model.errors.password}}
+      @isInvalid={{@model.errors.password}}
+      @type='password'
       autocomplete='new-password'
```

Comment:
What is the purpose of this attribute? and what is the expected beahviour of using this attribute?

Reply:
@calcaide this was previously part of the last input field, but the attribute is a hint to browsers to prevent autofilling the input field with a saved password.